ci/jenkins/test.sh

```diff
@@ -32,6 +32,10 @@ GO_VERSION=$(head -n1 "${WORKSPACE}/build/images/deps/go-version")
 IMAGE_PULL_POLICY="Always"
```

Comment:
Is it possible to write a new script for MC tests instead of modifying this one?

Reply:
okay , I will try to refactor the script.